Tasks/ServiceFabricDeploy/task.json

```diff
@@ -15,9 +15,9 @@
         "DeploymentGroup"
     ],
     "version": {
-        "Major": 1,
-        "Minor": 4,
-        "Patch": 7
+        "Major": 2,
```

Comment:
Given that the feature is backward compatible, I think we can go ahead with a minor version change.